dash/dependencies.py

```diff
@@ -1,18 +1,18 @@
-# pylint: disable=old-style-class, too-few-public-methods
+# pylint: disable=too-few-public-methods
```

Comment:
As a side-note, after these changes there are 55 instances of rules being disabled for a line -- wondering if (a) they could be disabled as a configuration, (b) if we're being heavy fisted with disables.